src/test/java/com/codeborne/selenide/webdriver/ChromeDriverFactoryTest.java

```diff
@@ -44,9 +44,7 @@ void defaultChromeOptions() {
     Capabilities chromeOptions = factory.createCapabilities(config, browser, proxy, browserDownloadsFolder);
     Map<String, Object> prefsMap = getBrowserLaunchPrefs(ChromeOptions.CAPABILITY, chromeOptions);
 
-    assertThat(prefsMap).hasSize(3);
-    assertThat(prefsMap).containsEntry("credentials_enable_service", false);
-    assertThat(prefsMap).containsEntry("plugins.always_open_pdf_externally", true);
+    assertThat(prefsMap).hasSizeGreaterThan(0);
```

Comment:
I don't think it's a good change. 
The name of test is `defaultChromeOptions()`, and its intent is exactly to test those default options.

Reply:
I disagree. Unit tests should test behaviour and not the implementation details. Testing on implementation details is considered to be an anti-pattern, because it slows down the refactoring and just re-assuring that the concrete code lines exist in the docs.

We could argue for deleting this test all together, but these checks are really binding to the implementation.

Reply:
That change could be interpreted: we assert that some preferences are actually set, and that one concrete folder is setup for a certain property - this is at least behavour-ish, imho.

Reply:
Yes, unit tests should test behaviour and not the implementation details. 
But in this case, it's _the behaviour_. Method name is `createCapabilities`, test name is `defaultChromeOptions` - it's all about generating those capabilities. This test tests exactly that.